darts/models/forecasting_model.py

```diff
@@ -24,29 +24,33 @@ class ForecastingModel(ABC):
     """ The base class for all forecasting models.
 
     All implementations of forecasting have to implement the `fit()` and `predict()` methods defined below.
-    """
 
+    Attributes
+    ----------
+    training_series
+        reference to the `TimeSeries` used for training the model in univariate cases.
+    covariate_series
+        reference to the `TimeSeries` used as covariate to train the model in multivariate cases.
+    target_series
+        reference to the `TimeSeries` used as target to train the model in multivariate cases.
+    """
     @abstractmethod
     def __init__(self):
         # Stores training date information:
         self.training_series: TimeSeries = None
+        self.covariate_series: TimeSeries = None
+        self.target_series: TimeSeries = None
 
         # state
         self._fit_called = False
 
     @abstractmethod
-    def fit(self, series: TimeSeries) -> None:
+    def fit(self) -> None:
         """ Fits/trains the model on the provided series
 
-        Parameters
-        ----------
-        series
-            the training time series on which to fit the model
+        Implements behavior that should happen when calling the `fit` method of every forcasting model regardless of
+        wether they are univariate or multivariate.
         """
-        raise_if_not(len(series) >= self.min_train_series_length,
```

Comment:
As far as I can see this is currently not being tested for the univariate model.

Reply:
good catch I moved this check in the parent class